vagrant_support/cron.yml

```diff
@@ -0,0 +1,45 @@
+- name: set cron mailto
+  cron:
+    env: yes
+    name: MAILTO
+    value: vagrant@localhost
+
+- name: cron remove-idle-group-members
+  cron:
+    hour: 7
+    minute: 0
+    job: 'cd /vagrant; ./scripts/remove_idle_group_members.pl'
```

Comment:
If this were bash, I would be certain that changing ```;``` to ``` && ``` is the right thing to do here. However, it's not. What happens if the cd fails, and the user has a $HOME/scripts/ with that in it?